server/src/game/server.rs

```diff
@@ -20,7 +20,15 @@ pub struct WsServer {
     games: HashMap<String, Game>,
 }
 
-impl WsServer {}
+impl WsServer {
+    /// Creates a new [`WsServer`].
+    pub fn new() -> Self {
```

Comment:
This function can be rewritten as `Self::default()` by adding `#[derive(Default)]`, see https://rust-lang.github.io/rust-clippy/master/#new_without_default